.rubocop.yml

```diff
@@ -0,0 +1,25 @@
+AllCops:
+  TargetRubyVersion: 2.6.9
+  NewCops: enable
+  Exclude:
+    - 'lib/generators/**/*'
+    - 'tmp/**/*'
+    - 'spec/**/*'
```

Comment:
excluding specs for the first cut to ensure rubocop related changes in spec directory don't influence changes aren't breaking in nature.